README.md

```diff
@@ -1 +1,182 @@
-# Hello-Microverse
\ No newline at end of file
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Hello Microverse README</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ ](#faq)
+- [📝 License](#license)
+
+
+
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+ **Hello Microverse** a simple assigment for Microverse to  test the use of linters and github.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Presentation</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+ The following are key features that are displayed on the portfolio
+
+- **headers**
+- **paragraphs**
+- **html** 
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to install stylelint as follows:
+
+
+```sh
+ npm init stylelint
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git https://github.com/AsciencioAlex/Hello-Microverse.git
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd Hello-Microverse
+  
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npx stylelint **.css
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  google-chrom index.html
+```
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  http://localhost/Hello-Microverse:8080/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Alex Asciencio**
+- GitHub: [@githubhandle](https://github.com/Asciencioalex)
+- Twitter: [@twitterhandle](https://twitter.com/Codnetech)
+- LinkedIn: [LinkedIn](https://linkedin.com/alex-asciencio-413612b9/)
+
```

Comment:
- [ ] As mentioned by the previous reviewer  to provide working links for twitter and linkedin, kindly ensure that you update them and use the working ones. Right now the twitter link opens a profile different to yours and the linkedIn one opens a broken page. If you don't have accounts I **suggest you create ones and use working links** or **you can remove them from now and add them them later when you will create accounts for each one**

Reply:
@tresorsawasawa the twitter account is mine not anyone's else.